source/declarations.tex

```diff
@@ -257,14 +257,33 @@
 is contextually converted to \keyword{bool} and
 the converted expression shall be a constant expression\iref{expr.const}.
 If the value of the expression when
-so converted is \tcode{true}, the declaration has no
-effect. Otherwise, the program is ill-formed, and the resulting
+so converted is \tcode{true}
+or the expression is evaluated in the context of a template definition,
+the declaration has no
+effect. Otherwise,
+%FIXME: What does it mean for a grammar term to fail? Also in [temp.res]/p6.
```

Comment:
Isn't this text the _definition_ of what it means for the static_assert-declaration to fail?

@jensmaurer, any thoughts? I think this seems clear?

Reply:
Yes, that's the intent: We defined "fails" here.

Note that the proposed resolution in CWG2518 marks "fails" in italics to highlight the fact this is a term definition.

Reply:
It's not in italics in this PR. Also, does https://github.com/cplusplus/draft/pull/6120#discussion_r1115069961, which made "int" into `int`, makes sense if the error comment is supposed to be a reproduction of the _string-literal_ to the `static_assert`?

Reply:
Oh, good catch, we need to italicise `fails`, thank you!